lib/razor/command/create_broker.rb

```diff
@@ -24,7 +24,7 @@ class Razor::Command::CreateBroker < Razor::Command
 Creating a simple Puppet broker:
 
     razor create-broker --name puppet -c server=puppet.example.org \\
-        -c environment=production --broker-type puppet
+        -c environment=production --broker_type puppet
```

Comment:
This is where underscores get kinda ugly: on the command line, words are usually separated by '-'. Since we will accept either, I'd prefer if the command line examples continue to use '-'